### ISSUE TITLE
Lab side bar categories don't make sense

### ISSUE THREAD

**Issue by mazzystr** [kubevirt/kubevirt.github.io#592]
/kind enhancement

Goal: Make categories make sense

Problem: Too wordy in a bad way

Why is this important: It's subconsciously obnoxious adding extra words on a label ie. "I need to go to the ATM machine".  Why say machine when machine is part of the acronym?  Our menu lives within the KubeVirt website.  "KubeVirt Upgrade" is not necessary.**

 

I suggest the following layout...

Kubernetes Platforms

    Minikube
    Kind
    OKD (page needs to be added)
    Cloud provided

Labs

    Using KubeVirt
    Using CDI

Can we automate the generation of this outline with tags?

Ref: [KNIECO-2252](https://issues.redhat.com/browse/KNIECO-2252)

**Comment by matthewcarleton** [comment]
@mindreeper2420 is this easy to automate?

**Comment by mindreeper2420** [comment]
@matthewcarleton Possibly? I'd have to take a deeper look into it. Page front matter would probably need to be added/updated, if it is not already specific enough.
